test/Common.cpp

```diff
@@ -78,7 +78,10 @@ std::optional<fs::path> findInDefaultPath(std::string const& lib_name)
 		fs::current_path() / ".." / "deps" / "lib",
 		fs::current_path() / ".." / ".." / "deps",
 		fs::current_path() / ".." / ".." / "deps" / "lib",
-		fs::current_path()
+		fs::current_path(),
+#ifdef __APPLE__
+		fs::current_path().root_path() / fs::path("usr") / "local" / "lib",
```

Comment:
If you do that this way, this will even prefer evmone in ``/usr/local/lib`` over a potential installation in ``/usr/lib``... is that sane behaviour :-)? Are there things like homebrew versions of evmone and those end up in ``/usr/local/lib`` or so, or how does this work in general :-)?
Just to make sure we've fully thought this through - it's a rather weird thing to do this here after all. But if the reasoning is sound we can merge this, of course.

Reply:
I mean, the same is the case for any linux installation - you don't have non-root rights to put stuff to ``/usr/lib``, even though (at least in some distributions) you may have rights for ``/usr/local/lib`` - yet we don't add ``/usr/local/lib`` to the default path there either... if you have stuff installed in any directory that's not in the default search path, I'd expect you to adjust the search path accordingly (in linux that'd be adding ``/usr/local/lib`` to ``LD_LIBRARY_PATH``, not sure how that's done in macos)... or alternatively you can just install it as user without messing with system installations at all in ``deps/``...

I mean, it's just testing, we can still merge this, if it makes your life easier :-D... but I find the reasoning a bit weak for it :-).